metrics/write.go

```diff
@@ -80,17 +86,21 @@ func SendRemoteWrite(config *ConfigWrite) error {
 	return c.write()
 }
 
-// Add the tenant ID header
+// Add the tenant ID header and custom headers
 func (rt *tenantRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
```

Comment:
I would not overload tenant RT in a hidden way - given custom headers gives more flexible let's remove tenant feature and expect users to pass custom header for tenant header AND rename those round tripper as `customHeaderRoundTripper` WDYT? (:

Reply:
👍🏽